Day1-Session2-Lecture/session2-dynamics-draft.ipynb

```diff
@@ -1021,7 +966,20 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.7"
+   "version": "3.7.10"
+  },
+  "toc": {
+   "base_numbering": 1,
+   "nav_menu": {},
+   "number_sections": true,
```

Comment:
@IAlibay maybe we could have just saved ourselves a bit of work by changing this metadata to False for this notebook only.